### ISSUE TITLE
UBERON:0001154 'vermiform appendix' - partonomy and taxon rule

### ISSUE THREAD

**Issue by paolaroncaglia** [obophenotype/uberon#1920]
**Uberon term**
UBERON:0001154 'vermiform appendix'

**Suggested revision and reasons**
Background: while working on terms needed for anatomograms for the Single Cell Expression Atlas, Mark Arends wrote to @sfexova:
"Based on anatomical and pathological textbooks ... and widespread usage in both anatomy and pathology, ...
Colon does _not_ include rectum, anal canal or _appendix_.
Large intestines (or large bowel) _includes_ colon, rectum, _appendix_, anal canal."
Based on that, and with support from @RDruzinsky too, in https://github.com/obophenotype/uberon/issues/1869#issuecomment-848896632 I edited 'vermiform appendix' so it isn't inferred to be part of colon. (The change isn't released yet, so not visible in OLS.)
However, 'vermiform appendix' is still "part of some lower digestive stract" rather than large intestine, and that axiom is asserted based on FMA.
We suggest to remove that axiom, replace it with "part of some large intestine", and add Mark's ORCID as a def dbxref.
Also, I think we could safely restrict 'vermiform appendix' taxonomically as 'present in taxon Mammalia', based on several references including the one that's already referenced in the homology notes of the term.
So basically we'd be saying that in those mammalian species that have an appendix, the appendix is part of the large intestine. I'll open a pull request.

**Comment by paolaroncaglia** [comment]
So with @RDruzinsky 's approval, I committed the changes for UBERON:0001154 vermiform appendix to be part of large intestine instead of lower digestive tract. Full diff [here](https://github.com/obophenotype/uberon/pull/1921/commits/cc47e70677a5e900376d3d58c1bf9be3ffc3b23e).
cc @sfexova 
Thanks
